packages/config/config/devices/0x0312/z97.json

```diff
@@ -0,0 +1,95 @@
+// NIE Technology Co., Ltd. ZW97 / Eva Logik
+// Outdoor Z-Wave Plug-In Module (2-Channel)
+// TODO: ZW97 support Over temperature protection, When the temperature is greater than 105 degrees Celsius, the Heat Alarm signal is sent out Notification Type : Heat Alarm 0x04 . Event :Overheat detected 0x02
+{
+	"manufacturer": "NIE Technology Co., Ltd.",
+	"manufacturerId": "0x0312",
+	"label": "ZW97",
+	"description": "Outdoor Z-Wave Plug-In Module (2-Channel)",
+	"devices": [
+		{
+			"productType": "0xc000",
+			"productId": "0xc007",
+			"zwaveAllianceId": 3701
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Outlet 1 (Left) State Notification",
+			"description": "When the state of Outlet 1 (Left) is changed, On (0xFF) or Off (0x00). The ZW97 will automatically send out a related command, On (0xFF) or Off (0x00) to its associated group.",
+			"maxNodes": 5
+		},
+		"3": {
+			"label": "Outlet 2 (Left) State Notification",
+			"description": "When the state of Outlet 2 (Right) is changed, On (0xFF) or Off (0x00). The ZW97 will automatically send out a related command, On (0xFF) or Off (0x00) to its associated group.",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"label": "LED Invert/Off",
```

Comment:
Change to "LED Indicator" for consistency with other files.

Reply:
Done